hack/util.sh

```diff
@@ -74,6 +74,7 @@ function configure_os_server {
 	openshift start \
 	--write-config=${SERVER_CONFIG_DIR} \
 	--create-certs=false \
+	--dns="tcp://${API_HOST}:53" \
```

Comment:
> What do you mean? The patch by Stephen worked when it was merged in the past, right?

It was never intended to work.  Arguments are ignored when specifying a config except where clayton wired up a few special cases.  Those special cases should die.